tasks/managed/update-cr-status/update-cr-status.yaml

```diff
@@ -29,26 +29,106 @@ spec:
         Path to the directory containing the result files in the data workspace which will be added to the
         resource's status
       type: string
+    - name: resultsDirName
```

Comment:
This isn't in the README. We need this when we already have resultsDirPath?

Reply:
I removed `resultsDirPath` and replaced it with `resultsDirName`. updated README.